apps/web/pages/org/[orgSlug]/[user]/index.tsx

```diff
@@ -17,17 +17,25 @@ export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
         not: null,
       },
       parent: {
-        slug: ctx.query.orgSlug as string,
+        OR: [
+          { slug: ctx.query.orgSlug as string },
+          {
+            metadata: {
+              path: ["requestedSlug"],
```

Comment:
Needed to make sure we also selected an unpublished organization to not end up in a 404 page